applications/endoscopy_tool_tracking/cpp/CMakeLists.txt

```
@@ -16,7 +16,7 @@
 cmake_minimum_required(VERSION 3.20)
 project(endoscopy_tool_tracking CXX)
 
-find_package(holoscan 0.5 REQUIRED CONFIG
+find_package(holoscan 2.1 REQUIRED CONFIG
```

For [NVAIE maintenance requirements](https://github.com/nvidia-holoscan/holohub/blob/main/doc/release.md#curated-backwards-compatible-applications) the `endoscopy_tool_tracking` application must be compatible with at least Holoscan SDK v1.0.3.

Note that there are some ongoing conversations and those requirements may change, but it would still be good practice to maintain backwards compatibility around endoscopy tool tracking in particular as it is a key Holoscan SDK example.

```suggestion
find_package(holoscan 1.0 REQUIRED CONFIG
```